styles/widgets/android5/treeView.android5.less

```diff
@@ -34,6 +34,10 @@
 }
 
 .dx-treeview {
+    .dx-treeview-search {
```

Comment:
This block can be used outside of `.dx-treeview` block. It will make selectors weaker

Reply:
Done